AmahiAnywhere/AmahiAnywhere.xcodeproj/project.pbxproj

```diff
@@ -783,14 +787,14 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = 325LAYFA25;
+				DEVELOPMENT_TEAM = 83F32NLU7E;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = AmahiAnywhere/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere;
+				PRODUCT_BUNDLE_IDENTIFIER = com.amahi.AmahiAnywhere.Marton.Test;
```

Comment:
Hi, @Marton-Zeisler! You forgot to bring back the old Bundle ID.

Reply:
Oh yes, apologies for that